test/e2e/networks/long.toml

```diff
@@ -0,0 +1,254 @@
+ipv6 = true
+initial_height = 2
+key_type = ""
+abci_protocol = "tcp"
+upgrade_version = "cometbft/e2e-node:local"
```

Comment:
Trying to run this locally (not the same manifest but with these parameters), it is complaining about the lack of `e2e-node:local`. I do have a `e2e-node:local-version` tag in my docker image list and that is also what you have in the Makefile so I don't know whether this should be changed here.

Reply:
Yes, it must be local-version, but it should be fixed now. Pls pull.